### ISSUE TITLE
Missing platform tag in wheel name

### ISSUE THREAD

**Issue by daskol** [tensorflow/tensorboard#6602]
TensorBoard from 2.12 up to now (2.14) does not start on Ubuntu 20.04 due to libc version mismatch (actual 2.32 vs expected 2.35). This should be explicitly specified in platform tag of a wheel in PyPI. I would be great at least to add this info in release notes explicitely.

```
TensorFlow installation not found - running with reduced feature set.
/usr/local/lib/python3.8/dist-packages/tensorboard_data_server/bin/server: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /usr/local/lib/python3.8/dist-packages/tensorboard_data_server/bin/server)
/usr/local/lib/python3.8/dist-packages/tensorboard_data_server/bin/server: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /usr/local/lib/python3.8/dist-packages/tensorboard_data_server/bin/server)
/usr/local/lib/python3.8/dist-packages/tensorboard_data_server/bin/server: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/local/lib/python3.8/dist-packages/tensorboard_data_server/bin/server)
```

**Comment by groszewn** [comment]
I believe this is related to #6578. The produced [tensorboard-data-server wheels](https://pypi.org/project/tensorboard-data-server/#files) do specify [manylinux2014](https://github.com/tensorflow/tensorboard/blob/65c03827602141928bb84f441e6c402c05ffeb32/tensorboard/data/server/pip_package/build.py#L41), but the [tensorboard wheels](https://pypi.org/project/tensorboard/#files) do not.

**Comment by daskol** [comment]
Yes, indeed but the version of glibc should be indicated explicitly as a platform tag in wheel according to Python's PEPs.

**Comment by groszewn** [comment]
The `tensorboard` (and `tb-nightly`) wheels don't require glibc, an underlying dependency on a separate wheel (i.e. `tensorboard-data-server`) is what requires glibc. Tensorboard itself should still run, just without the functionality in the separate `tensorboard-data-server` wheel (i.e. it falls back to the native python implementation).

#6578 tracks that the `tensorboard-data-server` wheel is bundled with a too-new version of glibc given its platform tag, which we're working on rectifying.

**Comment by groszewn** [comment]
I just released [tensorboard-data-server==0.7.2](https://pypi.org/project/tensorboard-data-server/#files) which should resolve this (i.e. using the `manylinux_2_31_x86_64`).
